src/arch/cuda/hybridAPI/buddy_allocator.cpp

```diff
@@ -0,0 +1,195 @@
+#include "buddy_allocator.h"
```

Comment:
Can you add a high level summary of how the buddy allocator works?

Reply:
Will do